nucypher/blockchain/eth/trackers/dkg.py

```diff
@@ -164,6 +167,10 @@ def web3(self):
     def contract(self):
         return self.coordinator_agent.contract
 
+    @property
+    def active_rituals(self) -> Dict[Tuple[RitualId, PhaseId], AsyncTx]:
+        return self.__phase_txs
```

Comment:
When do stale entries get cleared? Eg. ritual completed?

Reply:
As of the latest commit in this PR, these are not cleared.  I think the most straight-forward solution is to prune final asynx transactions on each round performance.  I'd elect to accept those changes in a subsequent PR, if that's acceptable.

Reply:
opened https://github.com/nucypher/nucypher/issues/3454